catalog/spatial/geocoding/spatial-geocoding-lucene/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -28,6 +28,21 @@
         <property name="indexLocation" value="data/geonames-index"/>
     </bean>
 
+    <reference id="geoExtractor" interface="org.codice.ddf.spatial.geocoding.GeoEntryExtractor"/>
+
+    <bean id="executorService" class="java.util.concurrent.Executors"
+          factory-method="newSingleThreadExecutor">
```

Comment:
Just a reminder comment to make sure we clean up this executor/single thread after we are done with initialization.

Reply:
Good call. Added executor.shutdown() at the end of the init() method